docs/PytorchXLA_overview.md

```diff
@@ -0,0 +1,270 @@
+### **Objective:**
```

Comment:
nit, can we change the file name to `pytorch_xla_overview.md` and add a title within the Doc Like `PyTorch/XLA Overview`

Reply:
You can take https://github.com/pytorch/xla/blob/master/docs/dynamo.md as an example

Reply:
Changed the filename and added title "Beginner's Guide to PyTorch/XLA" per @AlexWertheim suggestion